test/unittests/parser_test.cpp

```diff
@@ -312,6 +264,13 @@ TEST(parser, type_section_functype_out_of_bounds)
     EXPECT_THROW_MESSAGE(parse(wasm), parser_error, "Unexpected EOF");
 }
 
+TEST(parser, import_section_empty)
+{
+    const auto bin = bytes{wasm_prefix} + make_section(2, make_vec({}));
+    const auto module = parse(bin);
+    EXPECT_EQ(module.typesec.size(), 0);
```

Comment:
I guess you intended to check `importsec` here.
The tests below check `typesec`, too

Reply:
Good catch.

Reply:
Moved to #194